### PR TITLE
Add question of why add custom Gradle task like document, but Gradle can't execute?

### PR DESCRIPTION
**Pull request by helloworlde**
Add question of why add custom Gradle task like document, but Gradle can't execute?

### CODE REVIEW

**Comment by sriv** [comment]
@helloworlde, thank you for the contribution.

I would like to propose some minor edits, is it ok with you if I do?

**Comment by helloworlde** [comment]
Of course you can, thanks

File: faqs.rst [hunk 1]
```diff
@@ -84,6 +84,29 @@ The upper limit to the number of open files is low.
 Increase the upper limit by adding a command ``ulimit -S -n 2048`` to you ~/.profile.
 Don't forget to re-login for the changes to take effect.
 
+Why I add custom gradle task like document, but gradle can't execute?
```
Comment (sriv) [inline faqs.rst]: "Why can't gradle execute gauge specs, despite adding a task as per the document?"

File: faqs.rst [hunk 2]
```diff
@@ -84,6 +84,29 @@ The upper limit to the number of open files is low.
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: console
+
+    * What went wrong:
+    A problem occurred evaluating root project 'INDYGauge'.
+    > Could not get unknown property 'GaugeTask' for root project 'INDYGauge' of type org.gradle.api.Project.
+
+That's because Gradle can't recognize what GaugeTask is, you need change to it's qulified name like this
```
Comment (sriv) [inline faqs.rst]: This can happen when gradle is unable to recognize the GaugeTask. Try using the fully qualified name for GaugeTask as below:

Comment (helloworlde) [inline faqs.rst]: Look more standard, thanks

Comment (sriv) [inline faqs.rst]: Thanks. We have two options, you can edit and push, or I can merge and do the edits. Either way is fine, please let me know.

Comment (helloworlde) [inline faqs.rst]: Please merge it, thanks

**Pull request merged**